### PR TITLE
Create User Guide Article Holders 

### PR DESCRIPTION
**Pull request by hpdang**

### CODE REVIEW

File: website/sidebars.json [hunk 1]
```diff
@@ -8,6 +8,11 @@
     ],
     "User Guide": [
       "user-guide/user-guide-start",
+      "user-guide/user-guide-join-jitsi-meeting",
+      "user-guide/user-guide-start-a-jitsi-meeting",
+      "user-guide/user-guide-start-a-jitsi-meeting",
```
Comment (luixxiul) [inline website/sidebars.json:13]: duplicate?

**Comment by luixxiul** [comment]
Would you please explain what the intention of this PR is? Thanks!

**Comment by hpdang** [comment]
As there is not yet a structure on how user document should look like, it is difficult for people to know where to start, I look at different materials, I think people can only start if there is some guidance of what is needed. If the list of article ideas are accepted, I will start to work on them, it is better to inform the project what I want to work on.

**Comment by saghul** [comment]
> As there is not yet a structure on how user document should look like, it is difficult for people to know where to start,

Exactly! Thanks for doing this!

**Comment by saghul** [comment]
Can you please rename the md files so they contain no spaces?

**Comment by hpdang** [comment]
yes, changing them and also correct link on sidebar file, is it ok to start with those articles. I can create the content accordingly

**Comment by saghul** [comment]
>  is it ok to start with those articles?

Sure, any addition to the handbook, however small, is very welcome!

**Comment by luixxiul** [comment]
>  As it is written on the ReadMe page, it says every single small contribution is welcome. I am surprised by your comment.

I'm sorry for being jerk yet please understand too that I could not understand your intention because there is no word on the first post: https://github.com/jitsi/handbook/pull/28#issue-424384640

**Comment by hpdang** [comment]
@luixxiul no problem :), and you are right I should have put the PR description. Will do it next time. Can this PR be merged now?

**Comment by luixxiul** [comment]
Created from #32 to #36 to track the progress after this PR.

**Pull request merged**